andela/workdey
# Bg inactive facebook sign in #120271691

# #[120271691] Fixing of inactive facebook sign in button
#### What does this PR do?

This PR is a fix for the damaged facebook sign in link on the Workdey app
#### Description of Task to be completed?

The tasks completed include registering the app on facebook and getting the app_id and app secret.
These tokens are then listed in an `application.yml` file and the environment variables are set in the omniauth.rb initializer file. This will also be updated in production.
#### How should this be manually tested?
- Visit the [Workdey website](workdey-staging.herokuapp.com)
- Click on SIGN IN
- Click on facebook and the authentication should happen seamlessly
- At this point, you will be directed back to the app
#### Relevant Pivotal Tracker stories

https://www.pivotaltracker.com/story/show/120271691
